zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/legacycacheapi/LegacyCacheApiPayload.java

```diff
@@ -25,7 +25,7 @@ public LegacyCacheApiPayload(String collectionId, String uriToUpdate) {
         this.uriToUpdate = uriToUpdate;
     }
 
-    public void updatePublishDate(EventType eventType){
+    public void updatePublishDateToUnlockFromPublishing(EventType eventType){
```

Comment:
This still doesn't make sense to me. Are you updating the publishDate to unlock from Publishing? It looks like you're making the publishDate is null *if* the event is an UNLOCKED event.